### PR TITLE
revert(replays): Revert replay-video feature flag

### PR DESCRIPTION
**Pull request by cmanallen**
#skip-changelog

### CODE REVIEW

**Review by Dav1dde — APPROVED** [review]

**Comment by cmanallen** [comment]
Fixed without a revert!

**Comment by cmanallen** [comment]
Broken again thanks to some changes in getsentry.  I'm going to revert and re-assess how to manage this async.

File: CHANGELOG.md [hunk 1]
```diff
@@ -38,7 +38,6 @@
 - Add `cardinality_limited` outcome with id `6`. ([#3389](https://github.com/getsentry/relay/pull/3389))
 - Extract `cache.item_size` and `cache.hit` metrics. ([#3371]https://github.com/getsentry/relay/pull/3371)
 - Optionally convert segment spans to transactions for compatibility. ([#3375](https://github.com/getsentry/relay/pull/3375))
-- Add feature flag for replay video event types. ([#3402](https://github.com/getsentry/relay/pull/3402))
```
Comment (jjbayer) [inline CHANGELOG.md:41]: 24.4.0 was already released, so we should not remove features from the changelog for it.

Comment (jjbayer) [reply CHANGELOG.md:41]: (see https://github.com/getsentry/sentry/pull/69324#pullrequestreview-2014597929).

Comment (cmanallen) [reply CHANGELOG.md:41]: Should I open a new PR with the changelog entry?

Comment (jjbayer) [reply CHANGELOG.md:41]: Either that, or revert this revert and hard-code the feature flag to `true` in sentry (see other PR).

**Pull request merged**